lib/extras/statics.dart

```diff
@@ -0,0 +1,34 @@
+class Statics {
```

Comment:
Great to see those vars extracted to a separate location! My suggestion would be to rename it to `constants` instead of `statics`  for both class and file name, as it seems to me the first name is more widely used. What do you think?

Reply:
Thanks, good idea. Now it is fixed.